test/DependencyInjection.Tests/AspnetDiContainer/DependencyInjectionGrainTestsUsingASPNETDIContainer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Hosting;
+using Orleans.TestingHost;
+using TestExtensions;
+using UnitTests.Grains;
+
+namespace DependencyInjection.Tests.AspnetDIContainer
+{
+    [TestCategory("DI"), TestCategory("BVT")]
```

Comment:
I'd make these Functional instead, as they'll slow down the BVTs... But feel free to push back.
Also, I believe there is already another category for DI tests... maybe it was DependencyInjection? Don't remember, and I might be mistaken